server/update.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-BRANCH=`git branch|sed -ne 's,^\* ,,p'`
+BRANCH=$(git branch|sed -ne 's,^\* ,,p')
```

Comment:
Probably ```git branch --show-current``` is what you want to do.

Reply:
Updated as suggested. Tests passed, so merging.